.github/workflows/code-quality.yml

```
@@ -112,82 +118,59 @@ jobs:
+      - name: Ensure report directories exist
+        run: |
+          mkdir -p e2e-snapshots-main
+          mkdir -p e2e-snapshots-results
+          mkdir -p e2e-test-results
+          ls -la e2e-snapshots-results
+          ls -la e2e-test-results 
+          ls -la e2e-snapshots-main
+
+      - name: Start Apostrophe
+        run: docker compose up -d
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)